### ISSUE TITLE
Implement card specific CTA

### ISSUE THREAD

**Issue by maxvisser** [local-motion/product#357]
## Description

I think having these straight to the point action buttons will help people focus on what to do.

## CTA for each card

page | CTA |  amount | action
---|---|---|---
overview | Doe mee en meld je aan | unlimited | should go to login 
overview | Doe mee en ga aan de slag | unlimited | should go to active phase 
add-playground | Mijn speeltuin staat er niet bij | when no workspace id selected | change page
login | Maak een account | when unauthenticated | change page
add-team-member | Deel via e-mail | unlimited | send mail and log
flyer | Bestel flyers | 1 | send mail and set checkbox 'flyers are ordered'
flyer | flyers zijn uitgedeeld | 1 | set checkbox
meningen-inventariseren | Chat with teammembers | unlimited | change page
involve-admin | Send invite | 1 | send mail
commitment | Ons besluit staat vast | 1 | log
pick-date | Kies een geschikt moment | 1 | pick date and log
shout | Artikel in nieuwsbrief | 1 | set checkbox
shout | Plaats het op je website | 1 | set checkbox
Shout | Publiceer een persbericht | 1 | set checkbox
signonfence | Bestel bord | 1 | send mail and set checkbox
signonfence | Hang het bord bij de ingang van de tuin | 1 | set checkbox
signonfence | Pas je reglementen aan | 1 | set checkbox
signonfence | Publiceer nieuwe regels op je website | 1 | set checkbox
celebrate | Publiceer een persbericht | 1 | set checkbox
magnify | two cta's: rookvrij, niet rookvrij | multiple times | add checkmark depending on green or red button press

A page with multiple CTA's should always show the latest action; example:
if CTA: 'Flyers zijn besteld' is clicked the CTA should be 'Flyers zijn uitgedeeld'

## Definition of done

- [x] Go through every step and identify the CTA
- [x] Identify how often a action can be taken
- [x] Add right CTA to each card

**Comment by maxvisser** [comment]
Example of what I mean with moving CTA's:
![image](https://user-images.githubusercontent.com/10818702/55950030-3e13bd00-5c54-11e9-99b4-855e0bd85188.png)

**Comment by dimitriirtimid** [comment]
Ok, I am fine with the 'stuur uitnodiging' example above. Is clearly a better CTA that the previous version. 
For me the main thing is that all (or almost all) action are on team level and therefore synchronised. So if someone has completed a step (involving the manager, distributing flyers, etc.) then that step is completed for the whole initiative/group. So it makes no else to have the other group members also pressing buttons or ticking checkboxes for these steps.
Currently all checkboxes are synchronise so if one user ticks a box, all others will see that as well.
@maxvisser Do you agree with this?

**Comment by maxvisser** [comment]
Yes, I agree with this approach. For the user it still might feel weird though; so after implementing this. I would like to test if a user wants to check if a step is complete. as anyone could just check a checkbox or become a playground manager.

I can see this happening:
1. Max has ordered flyers
2. Max has spread the flyers
3. Dimitri wants to assist in spreading flyers
4. Dimitri has also spread flyers 

So basically CTA changes based on step 'status'; First CTA is order flyers, second spread flyers, for a user that did not order the flyers, he will get a CTA: I want to assist in spreading flyers; etc.

**Comment by dimitriirtimid** [comment]
Sound good.
I can image that some steps are individual and everybody gets his own checkbox. (Backend service is actually already prepared for this scenario.)
But imo we should stimulate co-operation within the team. So if I want to do (or challenge) something that you already did I should get in touch with you and not follow my own individual plan.
But I agree, let's see we the tests learn us.

**Comment by KarafiziArtur** [comment]
https://github.com/local-motion/onboarding-web/pull/415
